string trash_name
geometry_msgs/Point box_coords
